data/xml/2022.sigmorphon.xml

```diff
@@ -0,0 +1,301 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.sigmorphon">
+  <volume id="1" ingest-date="2022-07-06">
+    <meta>
+      <booktitle>19th SIGMORPHON Workshop on Computational Research in Phonetics, Phonology, and Morphology</booktitle>
```

Comment:
The title should be “Proceedings of the 19th…” (see [here](https://preview.aclanthology.org/naacl-22-ingestion/venues/sigmorphon/)).